CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
```

Comment:
Is there a reason why is false by default ?

Reply:
I was not sure, if it is guaranteed that every cluster you install the chart to has pod monitors. If so, I am happy to make it true by default.

Reply:
Oh yes we have them on all clusters greater than 18.2

Reply:
Well, most of them have pod monitors. It appeared around release 18 IIRC.
So I'd set this value as true by default, with the possibility of disabling for old releases.

But if you want to keep compatibility with old clusters, you may want to keep the service then, rather than completely disabling metrics?

Reply:
Workload clusters included? (As customers use the chart as well) Changing to default true then. 👍🏻

Reply:
KVM is frozen on an old MCB and therefor Flux release, so probably not an issue for really old ones.

Reply:
Yes, WCs > 18.2 have the prometheus-agent running, and support podmons.